config/base_boxes.yaml

```diff
@@ -76,3 +76,13 @@ boxes:
     box: centos7
     <<: *install_shell
     options: --install-type=devel
+
+  centos6-foreman:
+    box: centos6
+    <<: *install_shell
+    options: --install-type=foreman
+
+  centos7-foreman:
+    box: centos7
+    <<: *install_shell
+    options: --install-type=foreman
```

Comment:
I could look into doing that separately, as I am not sure the implications with other logic. An issue to track this would help.